frontend/src/components/LearnPageLayout.tsx

```diff
@@ -26,17 +27,19 @@ export interface LearnPageLayoutProps {
   header: string
   learnMoreHeader: string
   learnMoreLinks: ReadonlyArray<LearnMoreLink>
+  breadcrumbItems: BreadcrumbItemType[];
```

Comment:
Make it optional (undefined).
```suggestion
  breadcrumbItems?: BreadcrumbItem[];
```

Reply:
Fixed